modules/dynunet_pipeline/evaluator.py

```diff
@@ -147,8 +148,8 @@ def _compute_pred():
                 predictions = _compute_pred()
 
         inputs = inputs.cpu()
-        predictions = self.post_pred(predictions)
-        targets = self.post_label(targets)
+        predictions = torch.stack([self.post_pred(i) for i in decollate_batch(predictions)])
```

Comment:
no need `torch.stack()`.

Reply:
Here tensor is needed for the following calculation.

Reply:
I think you mean the `recovery_prediction` transform in this tutorial? Could you please also change it to `channel-first` transform?

Thanks.

Reply:
Hi Nic, these lines are inside the `engine`, thus they should still be `BCHWD` when input into `engine.state.output`, using post processing here is just to resample it back to the original size.